packages/nextjs/test/integration/test/server/doubleEndMethodOnVercel.test.ts

```diff
@@ -0,0 +1,18 @@
+import { NextTestEnv } from '../utils/server';
+
+// This test asserts that our wrapping of `res.end` doesn't break API routes on Vercel if people call `res.json` or
+// `res.send` multiple times in one request handler.
+//  https://github.com/getsentry/sentry-javascript/issues/6670
+it.skip('should not break API routes on Vercel if people call res.json or res.send multiple times in one request handler', async () => {
```

Comment:
This test randomly fails with `ERR_STREAM_WRITE_AFTER_END`, not sure if it's the testing environment or the case in #6099 still happens.